src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerDocumentManagementAmend.java

```diff
@@ -77,17 +78,9 @@ public void doConfigure(final ConfigBuilder<CaseData, State, UserRole> configBui
             .name("Document management: Amend")
             .description("Document management: Amend")
             .showSummary()
-            .grant(CREATE_READ_UPDATE, SUPER_USER,
-                ST_CIC_SENIOR_JUDGE, ST_CIC_CASEWORKER, ST_CIC_SENIOR_CASEWORKER,
-                ST_CIC_HEARING_CENTRE_ADMIN, ST_CIC_HEARING_CENTRE_TEAM_LEADER)
-            .grantHistoryOnly(
-                ST_CIC_CASEWORKER,
-                ST_CIC_SENIOR_CASEWORKER,
-                ST_CIC_HEARING_CENTRE_ADMIN,
-                ST_CIC_HEARING_CENTRE_TEAM_LEADER,
-                ST_CIC_SENIOR_JUDGE,
-                SUPER_USER,
-                ST_CIC_JUDGE)
+            .grant(CREATE_READ_UPDATE_DELETE, ST_CIC_SENIOR_CASEWORKER, ST_CIC_HEARING_CENTRE_TEAM_LEADER)
+            .grant(CREATE_READ_UPDATE, SUPER_USER, ST_CIC_SENIOR_JUDGE, ST_CIC_CASEWORKER, ST_CIC_HEARING_CENTRE_ADMIN)
+            .grantHistoryOnly(ST_CIC_JUDGE)
```

Comment:
Should ST_CIC_HEARING_CENTRE_ADMIN be grantHistory to be consistent with other PR?